.github/workflows/ruby.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       uses: actions/setup-node@v2
```

Comment:
Could also update setup-node and setup-ruby to latest versions if its easy enough to do.